sdks/python/apache_beam/io/gcp/experimental/spannerio_test.py

```diff
@@ -499,6 +499,7 @@ def test_batch_byte_size(
       # and each bach should contains 25 mutations.
       res = (
           p | beam.Create(mutation_group)
+          | 'combine to list' >> beam.combiners.ToList()
```

Comment:
Why do we need to perform this combining to run the test ?

Reply:
Yes, the `_BatchFn` requires a single iterable of collection and loop through them to make the batches. Just replicating the same pipeline for the batching in the `_WriteGroup` transform.

Reply:
Do users have to do this as well ? Seems like we are missing something in the implementation. How does Java implementation operate ?

Reply:
The user does not have to add ToList transform in the production pipeline. I only added this to test the batch process.
The previous implementation of batching (without ToList transform) was as per the java implementation but without the sorting of the transactions by table and primary key (this is also documented as a feature to be added later).

Reply:
Done!